lib/modulr/api/accounts_service.rb

```diff
@@ -3,13 +3,19 @@
 module Modulr
   module API
     class AccountsService < Service
-      def create(customer_id:, currency:, product_code:, options: {})
-        data = {
+      def find(id:)
+        response = client.get("/accounts/#{id}")
+        Resources::Accounts::Account.new(response, response.body)
+      end
+
+      def create(customer_id:, currency:, product_code:, **opts)
```

Comment:
Only kwargs from now on. This simplifies the calling code, as there is no need for optional hashes anymore. All required parameters are to be specified explicitly, and all other kwargs will be grouped automatically under the `opts` arg. 

So 

```rb
client.service.operation(required: 123, { optional: "ABC" })
```

It's now simply:

```rb
client.service.operation(required: 123, optional: "ABC")
```

Reply:
As we talked yesterday, it is not a problem which approach we choose, but as we talked, in my case it is easier to see which arguments are required if I see them as first level arguments and which ones are secondary inside the optional hash. As I said it is not a problem I'm also good with it 👍🏻